Automattic/woocommerce-payments
# Update deprecated interpolate-components to new @automattic/interpolate-components

Addresses public facing components in dependabot issue 28

#### Changes proposed in this Pull Request

The [`interpolate-components` npm package](https://www.npmjs.com/package/interpolate-components) has been deprecated and replaced with the [`@automattic/interpolate-components` package](https://www.npmjs.com/package/@automattic/interpolate-components). This PR removes the deprecated package and uses the new package, along with updating to version 1.2.1 from version 1.1.1.

#### Testing instructions

* JS tests should pass.
  * Be sure to run `npm install` locally.
* E2E tests should pass (at GitHub).
  * If doing this locally, be sure to `npm install && npm run build`.

-------------------


**Post merge**
